src/main/kotlin/org/wycliffeassociates/otter/common/data/rxmodel/Book.kt

```diff
@@ -0,0 +1,11 @@
+package org.wycliffeassociates.otter.common.data.rxmodel
+
+import io.reactivex.Observable
+
+data class Book(
+        val title: String,
+        val chapters: Observable<Chapter>,
+        val hasResources: Boolean,
```

Comment:
Should `Book` have a `resources` field? See https://github.com/WycliffeAssociates/otter-common/pull/95